platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/SyncTestUtilsTest.java

```diff
@@ -55,10 +55,10 @@ void testUnknownNonAncient() {
         knownSet.add(e3);
         knownSet.add(e1);
 
-        final GraphGenerations generations = new Generations(0, 1, 3);
+        final NonAncientEventWindow eventWindow = new NonAncientEventWindow(0, 1, 0, GENERATION_THRESHOLD);
```

Comment:
Is birthround testing relevant here?

Reply:
I'm leaning towards "not necessary". This is a test framework test, and the framework in question is something I'd like to deprecate in the near future.